ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -103,6 +103,19 @@ const DOTS_VARIANT_LIMIT = 5;
       marginLeft: theme.spacing.unit / 2,
       marginBottom: theme.spacing.unit / 2,
     },
+    previousPageArrow: {
+      marginLeft: -theme.spacing.unit,
```

Comment:
`theme.spacing.unit` is now `theme.spacing(1)` after the recent material-ui upgrade.